core/src/main/java/nl/nn/adapterframework/validation/XercesXmlValidator.java

```diff
@@ -132,7 +132,7 @@
 				cache.open();
 			} catch (ConfigurationException e) {
 				cache = null;
-				ConfigurationWarnings.add(log, "Could not configure EhCache for XercesXmlValidator (xmlValidator.maxInitialised will be ignored)", e);
+				ConfigurationWarnings.addGlobalWarning(log, "Could not configure EhCache for XercesXmlValidator (xmlValidator.maxInitialised will be ignored)", e);
```

Comment:
This can be a configuration warning

Reply:
it is in a static block where we don't have the configuration info